packages/docs/community/release-process/smart-contracts.md

```diff
@@ -137,13 +142,29 @@ Deploying a new contract release should occur with the following process:
     <td>
       <ol>
         <li>Confirm all contracts working as intended on Mainnet.</li>
+         <li>Update your forum post with the Mainnet governance outcome (<code>Passed</code> or <code>Rejected</code>) and notify the community in the Discord <code>#governance</code> channel.</li>
       </ol>
     </td>
   </tr>
 </table>
 
 If the contents of the release (i.e. source Git commit) change at any point after the release has been tagged in Git, the process should increment the release identifier, and process should start again from the beginning. If the changes are small or do not introduce new code (e.g. reverting a contract to a previous version) the audit step may be accelerated.
 
+### Communication guidelines
+
+Communicating your upcoming governance proposal to the community is critical and may help getting it approved. 
+
+Each smart contract release governance proposal should be accompanied by a [Governance category](https://forum.celo.org/c/governance/) forum post that contains the following information:
+* Name of proposer (individual contributor or organization).
+* Background information.
+* Link to the release notes.
+* Link to the audit report(s).
+* Anticipated timings for the Baklava and Alfajores testnets and Mainnet.
+
+It's recommended to post as early as possible and at minimum one week before the anticipated Baklava testnet governance proposal date. 
+
+Make sure to keep the post up to date. All updates (excluding fixing typos) should be communicated to the community in the [Discord](http://chat.celo.org/) `#governance` channel.
```

Comment:
Maybe there needs to be a way to request having the new governance proposal as part of the Celo mailing list for validators and stakeholders so we can add more spotlight on changes. I'd be careful though about not using the mailing list though to show that we are endorsing any of the proposals, more to show that they exist for the sake of validators and node operators.

Reply:
As far as I know there is no Celo Foundation mailing list. All conversations happen on the forum and Discord. My sense is that all community comms should happen there. cLabs however is starting a Engineering/Product newsletter in which some of this information will be communicated.